### PR TITLE
transmit a reporterId in reporter

### PR DESCRIPTION
**Pull request by laurgarn**

### CODE REVIEW

File: src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java [hunk 1]
```diff
@@ -153,7 +153,7 @@ public Flux<ModificationInfos> applyGroovyScript(UUID networkUuid, String varian
             getNetworkModificationInfos(networkUuid, variantId).flatMapIterable(networkInfos -> {
                 NetworkStoreListener listener = NetworkStoreListener.create(networkInfos.getNetwork(), networkUuid, groupUuid, networkModificationRepository, equipmentInfosService, false, networkInfos.isApplyModifications());
 
-                return execApplyGroovyScript(listener, groovyScript, reportUuid);
+                return execApplyGroovyScript(listener, groovyScript, reportUuid, groupUuid);
             })
         );
     }
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java:159]: You forgot to set groupUuid as taskKey for the reporter model in execChangeSwitchState ...

File: src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java [hunk 2]
```diff
@@ -1730,136 +1732,154 @@ public List<ModificationInfos> applyModifications(Network network, UUID networkU
...
+                        reportUuid, groupUuid);
+                    allModificationsInfos.addAll(modificationInfos);
+                }
+                break;
+
+                default:
+            }
+        } catch (PowsyblException e) {
+            NetworkModificationException exc = e instanceof NetworkModificationException ? (NetworkModificationException) e : new NetworkModificationException(MODIFICATION_ERROR, e);
+            ReporterModel reporter = new ReporterModel(NETWORK_MODIFICATION_REPORT_KEY, "Building node");
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java:1875]: groupUuid as taskKey ?

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java (hunk 1 above)
Comment (laurgarn) [reply src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java:159]: Done

File: src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java (hunk 2 above)
Comment (laurgarn) [reply src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java]: Done

**Review by FranckLecuyer — APPROVED** [review]

**Review by archendel — APPROVED** [review]
Code review : OK

**Comment by SlimaneAmar** [comment]
Remove the NETWORK_MODIFICATION_REPORT_KEY constant

**Review by SlimaneAmar — APPROVED** [review]

**Review by FranckLecuyer — APPROVED** [review]

**Review by FranckLecuyer — APPROVED** [review]

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]
remove also report added at the end in execSplitLineWithVoltageLevel method (log is done in the ConnectVoltageLevelOnLine apply method)

**Comment by SlimaneAmar** [comment]
> 

Done

**Pull request merged**